generated/nidaqmx/_task_modules/channels/ci_channel.py

```diff
@@ -12842,3 +12842,168 @@ def ci_velocity_meas_time(self):
             self._handle, self._name)
         check_for_error(error_code)
 
+    @property
```

Comment:
These commits are also weird... the deprecations aren't valid. `ci_count_edges_count_reset_reset_cnt` still exists.

Reply:
Latest commits fix it, woo!